### PR TITLE
Favicon added #114

### LINKED ISSUE #114: Add Favicon to the website
There is currently no favicon for the website. Add a favicon to the website. 
Favicon should be the logo of the Get Set FOSS.

Screenshot from 2019-11-10 14-21-18

### PR DESCRIPTION
**Pull request by yk1ax**
Closes #114
Favicon has been added

### CODE REVIEW

**Comment by yk1ax** [comment]
@blurry-x-face the issue is resolved, you can now review the PR

File: index.html [hunk 1]
```diff
@@ -10,10 +10,8 @@
     <title>Get Set FOSS!</title>
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
-    <!-- Icon -->
-    <link
-      href="https://cdn.lineicons.com/1.0.1/LineIcons.min.css"
-      rel="stylesheet"
```
Comment (dojutsu-user) [inline index.html]: Why did you remove this?

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

**Comment by dojutsu-user** [comment]
If a PR is solving an issue, write the PR description like this:
```
Closes #123

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris id dolor eu mi luctus accumsan. Sed tempus nulla eu metus imperdiet congue ac at ligula. Pellentesque bibendum, quam ut mattis ullamcorper, massa velit rhoncus nunc, id fermentum arcu sapien ut nibh.
```

File: index.html (hunk 1 above)
Comment (yk1ax) [reply index.html]: @dojutsu-user I thought this was like a failed link for the icon
I will revert this change

**Comment by yk1ax** [comment]
@dujutsu-user is it correct now ?

File: index.html [hunk 2]
```diff
@@ -10,11 +10,13 @@
     <title>Get Set FOSS!</title>
     <!-- Bootstrap CSS -->
     <link rel="stylesheet" href="assets/css/bootstrap.min.css" />
+    <!-- Favicon -->
+    <link rel="icon" href="assets/img/get-set-foss.ico" type="image/svg+xml"
```
Comment (dojutsu-user) [inline index.html]:
```suggestion
    <link rel="icon" href="assets/img/get-set-foss.ico" type="image/svg+xml" />
```

File: index.html [hunk 3]
```diff
@@ -10,11 +10,13 @@
...
+    />
```
Comment (dojutsu-user) [inline index.html]:
```suggestion
```

File: index.html [hunk 4]
```diff
@@ -10,11 +10,13 @@
...
     <!-- Icon -->
     <link
       href="https://cdn.lineicons.com/1.0.1/LineIcons.min.css"
-      rel="stylesheet"
-    />
+      rel="stylesheet"/>
```
Comment (dojutsu-user) [inline index.html]:
```suggestion
      rel="stylesheet"
    />
```

**Review by dojutsu-user — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (yk1ax) [reply index.html]: @dojutsu-user requested changes made in index.html

**Review by dojutsu-user — APPROVED** [review]
